opentdf-example.yaml

```diff
@@ -15,7 +15,7 @@ services:
     enabled: true
   authorization:
     enabled: true
-    url: http://localhost:8888
+    url: http://keycloak:8888
```

Comment:
does this work?  where is the setup for this?

Reply:
I added a `opentdf-dev.yaml` for local development. The readme should be updated. But yeah this will only work in docker compose setup like I documented.

Reply:
got it now.  You run a container that connects to the already running docker-compose network